contracts/protocol/tokenization/NTokenApeStaking.sol

```diff
@@ -81,19 +95,76 @@ abstract contract NTokenApeStaking is NToken {
         address receiverOfUnderlying,
         uint256[] calldata tokenIds
     ) external virtual override onlyPool nonReentrant returns (uint64, uint64) {
-        ApeStakingLogic.executeAutoWithdraw(
-            _ERC721Data.owners,
-            _apeCoinStaking,
-            POOL_ID(),
-            tokenIds,
-            from
-        );
+        for (uint256 index = 0; index < tokenIds.length; index++) {
+            ApeStakingLogic.executeUnstakePositionAndRepay(
```

Comment:
why don't we allow unstake a repay to take multiple token ids instead of calling this multiple times